services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/entity/decider/UsersEntityAccessInfo.java

```diff
@@ -15,6 +16,7 @@ public class UsersEntityAccessInfo {
 	Long entityId;
 	Long benefactorId;
 	boolean entityExists;
+	boolean hasUnmet;
 	AuthorizationStatus authorizationStatus;
```

Comment:
boolean -> List where the list is the IDs of the ARs that are unmet.  For non download cases the list is empty.